packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -310,6 +310,16 @@ export class SchemaPublisher {
       }
     }
 
+    let contextId: string | null = input.contextId ?? null;
+
+    if (
+      input.contextId === undefined &&
```

Comment:
```suggestion
      !contextId &&
```

maybe like this to prevent a situation where we pass an empty string or `null` accidentally (typescript is typescript)?

Reply:
I did an `undefined` check so that in case the c`ontextId` value sent from the client is null, no context fallback from github input is used. Not sure if really needed though

Reply:
https://github.com/kamilkisiela/graphql-hive/pull/3359/commits/66ca884fe98c59867d15fd26ca165c7ceddd0d26